public/api/vandal/css/app.59d17783.css

```diff
@@ -0,0 +1,8 @@
code.hljs.cs [class*=string],pre code.hljs.javascript [class*=string],pre code.hljs.json [class*=string]{color:#90ee90}pre code.hljs.coffeescript [class*=attribute],pre code.hljs.cs [class*=attribute],pre code.hljs.javascript [class*=attribute],pre code.hljs.json [class*=attribute]{color:#fff}pre code.hljs.coffeescript [class*=literal],pre code.hljs.cs [class*=literal],pre code.hljs.javascript [class*=literal],pre code.hljs.json [class*=literal]{color:#f99}pre code.hljs.coffeescript [class*=c1],pre code.hljs.cs [class*=c1],pre code.hljs.javascript [class*=c1],pre code.hljs.json [class*=c1]{color:#b4b4b4}pre code.hljs.coffeescript [class*=constant],pre code.hljs.cs [class*=constant],pre code.hljs.javascript [class*=constant],pre code.hljs.json [class*=constant]{color:#ffdf9d}pre code.hljs.coffeescript [class*=number],pre code.hljs.coffeescript [class*=nx],pre code.hljs.cs [class*=number],pre code.hljs.cs [class*=nx],pre code.hljs.javascript [class*=number],pre code.hljs.javascript [class*=nx],pre code.hljs.json [class*=number],pre code.hljs.json [class*=nx]{color:#88bfea}.overlay{background-color:transparent;opacity:1}.overlay,.top-level-contents{-webkit-transition:all .2s;transition:all .2s}.modal-dialog{max-width:60%!important}.modal{display:block;font-size:130%}.modal pre{margin-bottom:0}.modal code.hljs{margin-top:-40px;line-height:1.7rem}.modal.modal-leave-active{-webkit-animation:modal-close .2s;animation:modal-close .2s}.modal .modal-content{background-color:#2e3337;border:2px solid #000;max-height:1200px;overflow-y:scroll}.modal .modal-body{padding-top:0;margin-bottom:0}.modal .modal-header{padding-bottom:0;border-bottom:none}.modal .modal-header .close{z-index:9999}@-webkit-keyframes modal-open{0%{-webkit-transform:translateY(-400px) scale(.8);transform:translateY(-400px) scale(.8);opacity:0;max-height:0}to{-webkit-transform:translateY(0) scale(1);transform:translateY(0) scale(1);opacity:1;max-height:none}}@keyframes modal-open{0%{-webkit-transform:translateY(-400px) scale(.8);transform:translateY(-400px) scale(.8);opacity:0;max-height:0}to{-webkit-transform:translateY(0) scale(1);transform:translateY(0) scale(1);opacity:1;max-height:none}}@-webkit-keyframes modal-close{0%{-webkit-transform:translateY(0) scale(1);transform:translateY(0) scale(1);opacity:1;max-height:none}to{-webkit-transform:translateY(-400px) scale(.8);transform:translateY(-400px) scale(.8);opacity:0;max-height:0}}@keyframes modal-close{0%{-webkit-transform:translateY(0) scale(1);transform:translateY(0) scale(1);opacity:1;max-height:none}to{-webkit-transform:translateY(-400px) scale(.8);transform:translateY(-400px) scale(.8);opacity:0;max-height:0}}.modalIsOpen .overlay{position:fixed;top:0;bottom:0;left:0;right:0;background-color:#000;opacity:.55;z-index:1001}.modalIsOpen .top-level-contents{-webkit-filter:blur(1px);filter:blur(1px);opacity:.8;-webkit-transform:scale(.95);transform:scale(.95)}.modalIsOpen .modal{-webkit-animation:modal-open .2s;animation:modal-open .2s}
```

Comment:
Warum ist aller Vandal Code in Git eingecheckt? Müsste das nicht via Webpacker installiert und ausgeliefert werden? Können wir auch noch später machen, aber für Security-Updates ist der Ansatz mit Code ins Repo kopieren in 2022 keine gute Idee mehr.

Reply:
ja für den vandal ui client hatte es einfach einen generator der das Ganze so im public folder erstellt hat.
ich notiere mir mal das wir das später noch anschauen

Reply:
update: vandal ui ist jetzt raus. wir verwenden nur noch swagger welches ebenfalls eine sandbox bietet.